src/Tokenizer.js

```diff
@@ -29,6 +29,38 @@ function outputLink(cap, link, raw) {
   }
 }
 
+function indentCodeCompensation(raw, text) {
+  if (raw === undefined) {
+    return text;
+  }
+
+  const matchIndentToCode = raw.match(/^(\s+)(?:```)/);
+
+  if (matchIndentToCode === null) {
+    return text;
+  }
+
+  const indentToCode = matchIndentToCode[matchIndentToCode.length - 1];
```

Comment:
The spaces will always be at `[1]`

```suggestion
  const indentToCode = matchIndentToCode[1];
```

Reply:
it's more like using a magic number

Reply:
Does the first match at [0] not pick this up as well? If so can you just use `raw.exec` instead of `raw.match`?

Reply:
`[0]` will return the spaces and ` ``` ` not just the spaces. We could use the whole match if we do a look ahead `(?=```)` instead of a non-matching group `(?:```)` but I don't know if one is better than the other.